SAP/Script/FXVAirHandlingConfiguration.py

```diff
@@ -94,16 +94,14 @@
     page.WaitConfirm(1000)
   
   #Fan Motor RPM B
-  if air_handling_values["fan_motor_rpm_b"] != "SAP Defined" :
-   if air_handling_values["fan_motor_rpm_b"] != "Not Visible": 
+  if(air_handling_values["fan_motor_rpm_b"] != "SAP Defined" and  air_handling_values["fan_motor_rpm_b"] != "Not Visible"):
```

Comment:
Remove one space after 'and' there is a two space.

Reply:
I have removed the extra space.